nsv13/code/modules/overmap/camera.dm

```diff
@@ -46,7 +46,8 @@
 	user.click_intercept = src
 	if(position != "observer")
 		user.add_verb(overmap_verbs) //Add the ship panel verbs
-	if(mass < MASS_MEDIUM)
+	
+	if(mass < MASS_MEDIUM || !istype(src, /obj/structure/overmap/torpedo))
```

Comment:
should probably be like this if you want torpedoes to not return after that check. Otherwise, the check is basically always true.
```suggestion
	if(mass < MASS_MEDIUM && !istype(src, /obj/structure/overmap/torpedo))
```